src/agent/debuglet.ts

```diff
@@ -765,8 +765,8 @@ export class Debuglet extends EventEmitter {
             }
           ).debuggee.id;
           // TODO: Handle the case when `result` is undefined.
-          that.emit('registered', (result as {debuggee: Debuggee}).debuggee.id); // FIXME: Do we need this?
-          that.debuggeeRegistered.resolve(); // FIXME: Do we need this?
+          that.emit('registered', (result as {debuggee: Debuggee}).debuggee.id);
```

Comment:
It may make sense to add a comment here for either or both lines

Reply:
It's non-obvious at this code location, but the events are documented at the constructor and the promises are documented where they are declared.  I've added a comment at with the promises to help provide context for why they exist.